internal/pkg/db/redis/queries.go

```diff
@@ -128,7 +129,7 @@ func getObjectsByRange(conn redis.Conn, key string, start, end int) (objects [][
 // Return objects by a score from a zset
 // if limit is 0, all are returned
 // if end is negative, it is considered as positive infinity
-func getObjectsByRangeFilter(conn redis.Conn, key string, filter string, start, end int) (objects [][]byte, err error) {
+func getObjectsByRangeFilter(conn redis.Conn, key string, filter string, start, end int64) (objects [][]byte, err error) {
```

Comment:
See comment above. I don't know that int64 buys us anything here.

Reply:
Ditto to the above.

Reply:
Looks like it was propagation from the notification interface. I've corrected to an `int` implementaiton.